tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Datadog.Trace.ClrProfiler.IntegrationTests.csproj

```diff
@@ -39,6 +39,7 @@
   <ItemGroup>
     <PackageReference Include="Verify.Xunit" Version="14.13.1" />
     <PackageReference Include="DiffPlex" Version="1.7.2" />
+    <PackageReference Include="Xunit.Combinatorial" Version="1.6.24" />
```

Comment:
Could/should we add this everywhere do you think? e.g. asm/debugger etc may want to use it?

Reply:
Added it to the Directory.Props